### PR TITLE
create example-node

### PR DESCRIPTION
**Pull request by antonio-ivanovski**
This PR adds example implementation for CLI app based on Node

### CODE REVIEW

File: implementations/example-node/README.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+This is a example Node implementation CLI wrapper for an implementation. Once the CLI is in place, you can take the command and forward it to an JS library implementation or API endpoint.
```
Comment (TallTed) [inline implementations/example-node/README.md]:
```suggestion
This is a example Node implementation of a CLI wrapper for an _____ implementation. Once the CLI is in place, you can take the command and forward it to a JS library implementation or API endpoint.
```

File: implementations/example-node/README.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+## Running
+
+The wrapper exposes `validate` CLI command. If you want to try this locally outside of the Docker environment, navigate to this package `cd implementations/example-node` and run the following command:
```
Comment (TallTed) [inline implementations/example-node/README.md]:
```suggestion
The wrapper exposes the `validate` CLI command. If you want to try this locally outside of the Docker environment, navigate to this package (e.g., `cd implementations/example-node`), and run the following commands:
```

File: implementations/example-node/README.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+```bash
+npm install
+npm link
+```
+
+After this you will be able to run the implementation without the need for Docker
```
Comment (TallTed) [inline implementations/example-node/README.md]:
```suggestion
Subsequently, you will be able to run the implementation without needing Docker:
```

File: implementations/example-node/README.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+To run the validator using Docker:
+
+```bash
+docker run -v $(pwd)/tests:/tests example-node-validator validate \
+  --input /tests/input/valid-credential.json \
+  --config '{"check":"identifier"}' \
+  --output /tests/output/1-sample-impl.json
+```
+
+Make sure to mount the appropriate directories for input and output files.
```
Comment (TallTed) [inline implementations/example-node/README.md]:
~~~suggestion
To run the validator using Docker, mount the appropriate directories for input and output files, and issue a command like the following:

```bash
docker run -v $(pwd)/tests:/tests example-node-validator validate \
  --input /tests/input/valid-credential.json \
  --config '{"check":"identifier"}' \
  --output /tests/output/1-sample-impl.json
```
~~~

**Review by TallTed — CHANGES_REQUESTED** [review]
Small fixes, one question (marked by `____`)

File: implementations/example-node/README.md (hunk 1 above)
Comment (antonio-ivanovski) [reply implementations/example-node/README.md]: Something along these lines?

```suggestion
This is a example Node CLI wrapper that can help you integrate either:
- JS library implementation that you should add to the `dependencies` section of `package.json` and import it inside the `main.js`
- Framework agnostic server implementation that is accessible via HTTP API. You can use `fetch` to make the requests.
```

Comment (TallTed) [reply implementations/example-node/README.md]: Getting better... What do you think of this?
```suggestion
This is an example Node CLI wrapper that can help you integrate either:
- a JavaScript library implementation that you should add to the `dependencies` section of `package.json` and import inside the `main.js`.
- a Framework-agnostic server implementation that is accessible via HTTP API. You can use `fetch` to make the requests.
```

Comment (antonio-ivanovski) [reply implementations/example-node/README.md]: Looks good to me 5ebc557

**Comment by thereisnogabe** [comment]
@antonio-ivanovski thanks for the work! would you mind transferring this PR to https://github.com/decentralgabe/vc-test-suite-template

**Pull request closed without merging**